### PR TITLE
Add Naver site verification key for linkwikiwiki (T6764)

### PR DESCRIPTION
**Pull request by R4356th**
Adds Naver site verification key for linkwikiwiki per T6764. The wiki is private so I had to copy the DB name from a custom domain request.

### CODE REVIEW

**Review by RhinosF1 — COMMENTED** [review]
The example above seems to use a different hook. Is there any difference between the two? Should one be changed?

**Comment by Universal-Omega** [comment]
> The example above seems to use a different hook. Is there any difference between the two? Should one be changed?

I think this one is the better one in this case (the one R4356th used)

**Comment by R4356th** [comment]
The above example is pretty odd as it is not really using the ``ParserOutput`` hook.

**Comment by RhinosF1** [comment]
I stole it off a MediaWiki.org example when I wrote that.

**Comment by R4356th** [comment]
> I stole it off a MediaWiki.org example when I wrote that.

Okay then maybe consider reading the docs for ``OutputPage`` or reading the example of Meta if you are still uncomfortable.

File: LocalWiki.php
Comment (Universal-Omega) [inline LocalWiki.php:180]: Will cause errors if this hook is not also changed with removal of param in function

File: LocalWiki.php
Comment (Universal-Omega) [inline LocalWiki.php:181]: Function should be renamed to for new hook

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: LocalWiki.php
Comment (R4356th) [reply LocalWiki.php]: Done.

**Review by Universal-Omega — APPROVED** [review]

**Comment by Reception123** [comment]
@Universal-Omega since you approved, is this waiting for anything else before it should get merged?

**Comment by Universal-Omega** [comment]
> @Universal-Omega since you approved, is this waiting for anything else before it should get merged?

Nope I just haven't had PC access in 2 days. I will today and will merge

**Pull request merged**
